captcha.js

```diff
@@ -1,45 +1,42 @@
-const Canvas = require('canvas');
+const Canvas = require('canvas')
+const fs = require('fs')
 let randomText = (length) => {
-    let str = '';
-    var range = 'abcdefghijklmnopqrstuvwxyz0123456789';
-    for(i = 0; i < length; i++) {
-        str += range.charAt(Math.floor(Math.random() * range.length));
+    let str = ''
+    var range = 'abcdefghijklmnopqrstuvwxyz0123456789'

```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nidrux/simple-captcha-generator/pullRequest?prid=7734724)
```suggestion
    var range = "abcdefghijklmnopqrstuvwxyz0123456789"
```